recipe-ts-runtime/src/AbstractOven.ts

```diff
@@ -0,0 +1,7 @@
+import { Cake } from "./Cake";
+
+export abstract class AbstractOven {
+    protected createCake(other?: Cake): Cake {
```

Comment:
Any reason for this? Will it be overridden at some point? Otherwise could make it static or just invoke `new Cake` 🤷‍♂️

Reply:
Yup Allocadia makes use of this. See `AllocadiaCake`. It's a subclass that allows de-serialization of string ids into object ids.